docs/docs-components/data/components.js

```diff
@@ -336,6 +336,34 @@ const componentData: $ReadOnlyArray<ComponentData> = [
       },
     },
   },
+  {
+    id: 'BannerUpsell',
+    platform: {
+      web: {
+        name: 'BannerUpsell',
+        visual: {
+          svg: <BannerUpsell />,
+        },
+        alias: ['Banner', 'Offer Banner', 'CTA Banner'],
```

Comment:
add Upsell as an Alias so it can pickup previous commits

Reply:
DONE